packages/replay/test/unit/index-errorSampleRate.test.ts

```diff
@@ -54,7 +54,7 @@ describe('Replay (errorSampleRate)', () => {
     expect(replay).not.toHaveLastSentReplay();
 
     captureException(new Error('testing'));
-    jest.runAllTimers();
+    jest.advanceTimersByTime(5000);
```

Comment:
I had to replace a couple of `runAllTimers` calls with `advanceTimersByTime` because apparently, two timers don't seem to work with `runAllTimers`. I'm honestly not entirely sure why and I don't fully understand what might be causing these errors but I _think_ correctness-wise, the debounce function isn't responsible for the test fails.  

WDYT, does this change make sense? @billyvg you know these tests best. Is there something I'm missing?

Reply:
Not sure, I would think `runAllTimers` should work -- in any case I think replacing it with `advanceTimersByTime` is fine. 

Should we replace `5000` (and the default for `flushMinDelay`) with a constant?

Reply:
My theory is that the cancellation of the two timers when one of them is invoked might somehow interfere with whatever `runAllTimers` is doing but tbh this is a guess at best.

> Should we replace 5000 (and the default for flushMinDelay) with a constant?

Sure can do

Reply:
> Sure can do

On second thought, let's do this in a separate PR, just to not clutter this one. I've checked and we're using 5000 quite a lot in tests.